data/bibliography.json

```diff
@@ -1387,7 +1387,7 @@
       "container-title": "ACM SIGART Bulletin",
       "page": "31–32",
       "issue": "77",
-      "abstract": "The Interslip-D project was formed to develop a personal machine inplementation of Interlisp for use as an environment for research in artificial intelligence and cognitive science [Burton et al., 80b]. This note describes the principal developments since our last report almost a year ago [Burton et al., 80a].",
+      "abstract": "The Interslip-D project was formed to develop a personal machine implementation of Interlisp for use as an environment for research in artificial intelligence and cognitive science [Burton et al., 80b]. This note describes the principal developments since our last report almost a year ago [Burton et al., 80a].",
```

Comment:
Interlisp-D, but this and the "inplementation" are in the document on the ACM DL.  I have not found a scanned image of the Bulletin, so I don't know whether these are transcription errors or actually present in the original text.